android/persistkit/src/main/java/co/omise/persistkit/Crypter.kt

```diff
@@ -21,8 +22,6 @@ private var forceCreateKeyStore: Boolean = false
 open class Crypter(private val aliasKeyName: String) {
     lateinit var keyStore: KeyStore
 
-    private val cipher: Cipher by lazy { Cipher.getInstance(TRANSFORMATION_SYMMETRIC) }
```

Comment:
Was there any particular reason for moving the cipher into the functions? 🤔

Reply:
I found a few issues earlier about thread safety. The gist was that there's a small chance that the encode and decode methods are called at almost the same time and they used the same cipher instances

Reply:
okk fair enough 👍